### PR TITLE
add peepolove cmd + image arg

### PR DESCRIPTION
**Pull request by gc**

### CODE REVIEW

File: src/commands/Fun/peepolove.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+			description: language => language.tget('COMMAND_SHINDEIRU_DESCRIPTION'),
+			extendedHelp: language => language.tget('COMMAND_SHINDEIRU_EXTENDED'),
+			requiredPermissions: ['ATTACH_FILES'],
+			runIn: ['text', 'dm'],
+			usage: '<image:image>'
+		});
+	}
+
+	public async run(message: KlasaMessage, [imageBuffer]: [Buffer]) {
+		console.log(imageBuffer);
```
Comment (kyranet) [inline src/commands/Fun/peepolove.ts]: 🧹
```suggestion
```

File: src/commands/Fun/peepolove.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+				type: 'round',
+				radius: 330 / 2,
+				restore: true
+			})
+			.rotate(0.4)
+			.addImage(this.handsImage!, 0, 0)
+			.toBufferAsync();
+
+		const finishedImage = new MessageAttachment(await imageBuff, `peepoLove.png`);
+
```
Comment (kyranet) [inline src/commands/Fun/peepolove.ts]: 🧹
```suggestion
```

**Review by kyranet — CHANGES_REQUESTED** [review]

**Review by kyranet — DISMISSED** [review]

File: src/commands/Fun/peepolove.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+export default class extends SkyraCommand {
+
+	private bodyImage: Buffer | null = null;
+	private handsImage: Buffer | null = null;
+
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			bucket: 2,
+			cooldown: 10,
+			description: language => language.tget('COMMAND_SHINDEIRU_DESCRIPTION'),
```
Comment (favna) [inline src/commands/Fun/peepolove.ts]: Wow I was under the impression this was `peepolove`, not `SHINDEIRU`

File: src/commands/Fun/peepolove.ts [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+			extendedHelp: language => language.tget('COMMAND_SHINDEIRU_EXTENDED'),
```
Comment (favna) [inline src/commands/Fun/peepolove.ts]: ditto

**Review by favna — CHANGES_REQUESTED** [review]
And also add the actual translations please

**Review by favna — APPROVED** [review]

**Review by kyranet — APPROVED** [review]

**Pull request merged**